src/libraries/System.IO.FileSystem.AccessControl/ref/System.IO.FileSystem.AccessControl.cs

```diff
@@ -118,4 +120,37 @@ public abstract partial class FileSystemSecurity : System.Security.AccessControl
         public void SetAccessRule(System.Security.AccessControl.FileSystemAccessRule rule) { }
         public void SetAuditRule(System.Security.AccessControl.FileSystemAuditRule rule) { }
     }
+    public class MemoryMappedFileSecurity : System.Security.AccessControl.NativeObjectSecurity
```

Comment:
@JeremyKuhne I am going to have to add a couple of new classes: `MemoryMappedFileAccessRule` and `MemoryMappedFileAuditRule`. They did not use to exist in .NET Framework. Should we submit them for API review?

Reply:
After deeper investigation, I concluded that they will not be necessary.